zipkin-server/src/main/java/zipkin2/server/internal/brave/TracingConfiguration.java

```diff
@@ -73,20 +73,21 @@
     return ThreadLocalSpan.create(tracing.tracer());
   }
 
-  /** Controls aspects of tracing such as the name that shows up in the UI */
-  @Bean Tracing tracing(Reporter<Span> reporter, SelfTracingProperties config) {
-    final Sampler sampler;
+  @Bean Sampler sampler(SelfTracingProperties config) {
```

Comment:
exposed this for sanity reasons (I can look at the /actuate/beans endpoint to see which sampler is in use